test/tsconfig.json

```diff
@@ -3,5 +3,11 @@
   "include": [
     "../src/**/*",
     "**/*.ts"
-  ]
+  ],
+  "compilerOptions": {
+    "typeRoots": [
+      "../node_modules/@types",
+      "./ambient.d.ts"
```

Comment:
This adds the custom type file as a complier option so TypeScript understands how to resolve `*.json` modules in the tests.